### DISCUSSION TITLE
How can i add botguard ?

### DISCUSSION THREAD

**Discussion by 0xrotense** [YOURLS/YOURLS#3788] [General]
Please, How do i integrate botguard.net to my yourls.org URL shortner website, So it can block access on all the generated short URLs also ?

Thanks

**Comment by dgw** [comment]
As far as I can tell, this is a web server plugin, or a cloud protection service like Cloudflare. There's no direct integration with YOURLS.

**Reply by 0xrotense** [reply]
@dgw , So you recommend i integrate it on my server directly and not in the YOURLS right?

That means I will need a root access VPS and not cpanel right ?

Thanks & Regards.

**Reply by dgw** [reply]
I don't "recommend" anything, as I've never heard of this service. But their documentation talks about integrating with Apache or Nginx, and that's not something you can do through YOURLS itself.

**Reply by 0xrotense** [reply]
Alright, thanks for that.
